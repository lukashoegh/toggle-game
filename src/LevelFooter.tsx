import * as React from 'react';

class LevelFooter extends React.Component<null, null> {
    render() {
        return (
            <div>
                <button>Restart level</button>
            </div>
        );
    }
}

export default LevelFooter;