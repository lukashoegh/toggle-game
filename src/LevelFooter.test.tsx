import * as ReactTestRender from 'react-test-renderer';
import * as React from 'react';
import LevelFooter from './LevelFooter';

it('renders correctly', () => {
    const tree = ReactTestRender.create(
        <LevelFooter />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});