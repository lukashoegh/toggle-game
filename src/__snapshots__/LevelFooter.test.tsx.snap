exports[`test renders correctly 1`] = `
<div>
  <button>
    Restart level
  </button>
</div>
`;
